analyzer/src/main/kotlin/MavenSupport.kt

```diff
@@ -222,15 +222,13 @@ class MavenSupport(localRepositoryManagerConverter: (LocalRepositoryManager) ->
                     remoteArtifactCache.write(artifact.toString(), yamlMapper.writeValueAsString(it))
                 }
             } else {
-                if (Main.stacktrace) {
-                    artifactDownload.exception.printStackTrace()
+                artifactDownload.exception.message?.let {
+                    log.debug { it }
                 }
```

Comment:
I would prefer this to get a bit more detail when debug is enabled:

```
log.debug { "Could not find '$artifact' in '$repository': ${artifactDownload.exception.collectMessages()}"  }
```

Maybe also with joinToString("\n").

Reply:
Done.